katacomb/katacomb/conf/narrow_L.yaml

```diff
@@ -0,0 +1,58 @@
+# Default L-band MFImage configuration options for 'offline' mode
+#
+# These parameters are the defaults used by the `continuum_image.py` script for
+# offline re-processing. The offline parameters are optimised to produce the
+# best continuum image possible, without consideration for runtime or the
+# quality of CLEAN components for continuum subtraction.
+#
+# Further details of the parameter selection can be found in JIRA ticket
+# SPR1-879: https://skaafrica.atlassian.net/browse/SPR1-879
```

Comment:
This appears to be pointing at the wrong ticket, its the one for wide band not narrow.